policies/contractors-invoicing-policy.md

```diff
@@ -0,0 +1,46 @@
+# Invoicing Contractors Policy
+
+The OpenSSL organization is purely remote without any office. The organization has got only paid resources as contractors. Thus each contractor is obliged to submit their invoices against the OSS based on the contract. **The submission should be done no later than the "Submission due dates" in the table "Payment Run Schedule" below for the given invoicing period or the payment will be subject to the next run (likely next month)**.
+
+## Contractor Invoice Submission
+
+1. Contractors send their invoices via the form or send them to **finance@openssl.org** directly in the given format and information below. If an invoice is submitted via a form, notification to the finance@openssl.org mailing list will be sent automatically. In the form:
+   * Select the month for your invoice.
+   * Type the number of worked hours in the given month. If contractor has in their contract a fixed monthly rate, just type any number.
+   * Add invoice that should be in the given format and name: **[current fiscal year][date]-[month(s)][year].pdf**
+ > eg: **20220813-Jul2022.pdf** or **20220813-JunJul2022.pdf** for invoicing more months
+
+## Processing Submitted Invoices
+
+2. Manager ensures invoices are correct (where applicable - right amounts for days worked and the requested amount, invoice is within the 3-month window, 1800 worked hours limit, ...). 
+
+3. Correct invoices will be committed to the oss/financial/. repository with the right filename format
+
+4. Manager prepares the payments CSV file and commits to the repo as soon as there are all eligible and correct invoices in place for a given invoicing period. 
+
+   * The payments CSV batch file will be created and committed for payment no later than **Payment Run Date** from the table below.
+   * If there is an exception, requestor should send it to their manager and to the **finance@openssl.org** mailing list.
+
+5. Manager requests (TO'ing) the treasurer to make the batch payment along with the notice to finance@openssl.org (CC'ing).
+<br><br>
+
+* Additional expenses as approved can be included, subject to another process around making sure they are itemized in the right way for tax (etc). These additional expenses should be submitted via a google form that is created for this purpose. This will send a notification to the finance@openssl.org mailing list too.
+* Manager to batch payments (contractors expenses, invoices) as much as possible, to have a maximum of two potential payment dates a month, and communicate that to contractors.
+
+## Payment Run Schedule Table
```

Comment:
This should be an attachment/appendix/schedule.  It should **not** be part of the policy.

Reply:
Agreed.  I'm not even sure why we have that table in public, but fine.  I suggest placing it in the parallel directory `policy-supplemental`, like @TamaraDale did with #24

Reply:
OK, it sounds good